.ado/templates/prep-and-pack-nuget.yml

```diff
@@ -40,11 +40,32 @@ parameters:
     default: false
 
 steps:
+  # To be removed before merge (when new image is ready)
   - powershell: |
-      Write-Host "##vso[task.setvariable variable=basePlatform]${{ parameters.slices[0].platform }}"
-      Write-Host "##vso[task.setvariable variable=baseConfiguration]${{ parameters.slices[0].configuration }}"
-    displayName: Set base slice to ${{ parameters.slices[0].platform }}.${{ parameters.slices[0].configuration }}
-      
+      [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
+      Invoke-Expression "& { $(Invoke-RestMethod https://aka.ms/install-powershell.ps1) } -UseMSI -Quiet"
+    displayName: Install pwsh
```

Comment:
pwsh should already be installed, are you seeing something missing?

Reply:
Yes, it was showing as not installed on our new images. I added it as a dependency though, which I  think will have propagated by now